FirebaseVertexAI/Sources/Chat.swift

```diff
@@ -26,9 +26,29 @@ public class Chat {
     self.history = history
   }
 
+  private let historyLock = NSLock()
+  private var _history: [ModelContent] = []
```

Comment:
Have you considered the strategy you used in `VertexAI`? This one:
```
#if compiler(>=6)
  private nonisolated(unsafe) var _history: [ModelContent] = []
#else
  private var _history: [ModelContent] = []
#endif
```

I _think_ with the change above, if we were willing to mark `Chat` as `final`, it could be `Sendable` instead of `@unchecked Sendable`.

Reply:
Good point. Much better! Changed.